test/viewmodels/GuaranteeAccountViewModelSpec.scala

```diff
@@ -18,21 +18,44 @@ package viewmodels
 
 import config.AppConfig
 import connectors.AccountStatusOpen
-import models.{GeneralGuaranteeBalance, GuaranteeAccount}
+import models.{Amounts, DueDate, GeneralGuaranteeBalance, GuaranteeAccount, GuaranteeTransaction, TaxType, TaxTypeGroup}
 import play.api.test.Helpers
 import utils.SpecBase
 
-import java.time.LocalDateTime
+import java.time.{LocalDate, LocalDateTime, Month}
 
-class GuaranteeAccountViewModelSpec extends SpecBase  {
+class GuaranteeAccountViewModelSpec extends SpecBase {
   val guaranteeAccount = GuaranteeAccount("gan", "eori", AccountStatusOpen,
-                                                 Some(GeneralGuaranteeBalance(BigDecimal(10000.50), BigDecimal(5000.10))))
+    Some(GeneralGuaranteeBalance(BigDecimal(10000.50), BigDecimal(5000.10))))
+
+  val guaranteeAccount02 = GuaranteeAccount("gan", "eori", AccountStatusOpen,
+    Some(GeneralGuaranteeBalance(BigDecimal(0), BigDecimal(5000.10))))
 
   implicit val appConfig = mock[AppConfig]
-  
+
   val model = GuaranteeAccountViewModel(guaranteeAccount, LocalDateTime.parse("2020-04-08T12:30"))(Helpers.stubMessages())
+  val model02 = GuaranteeAccountViewModel(guaranteeAccount02, LocalDateTime.parse("2020-04-08T12:30"))(Helpers.stubMessages())
 
   "GuaranteeAccountViewModel" should {
+
+    "include the usedPercentage and usedFunds" in {
+      model02.account.balances.get.usedFunds mustBe BigDecimal(-5000.1)
+      model02.account.balances.get.usedPercentage mustBe BigDecimal(0)
+    }
+
+    "return security reasons from duedates" in {
+      val amt = Amounts("20.00", Some("30.00"), Some("10.00"), "2020-08-01")
+      val ttg = TaxTypeGroup(taxTypeGroup = "VAT", amounts = amt, taxType = TaxType("VAT", amt))
+      val dd = DueDate(dueDate = "2020-07-28", reasonForSecurity = Some("T24"), amounts = amt, taxTypeGroups = Seq(ttg))
+      val guranteeTxn = GuaranteeTransaction(LocalDate.of(2019, Month.OCTOBER, 23),
```

Comment:
Better to give a line break before this line to increase the readability. Same for line no 49 and 48